deny.toml

```diff
@@ -17,6 +17,8 @@ allow = [
     "BSD-3-Clause",
     "ISC",
     "MIT",
+    "MPL-2.0",
+    "OpenSSL",
```

Comment:
Why is OpenSSL license still necessary if https://github.com/bytecodealliance/wasmtime/pull/5929#issuecomment-1492628300 has switched to `rustls`?

Reply:
Yes, because the `ring` crate includes the OpenSSL license in its LICENSE file.